src/cloud/project/project-webint-snap.md

```diff
@@ -9,6 +9,9 @@ You can back up and restore specific environments at any time using a snapshot.
 
 The Pro Staging and Production environments receive regular backups for disaster recovery purposes by default, see [Pro Backup & Disaster Recovery]({{ site.baseurl }}/cloud/architecture/pro-architecture.html#backup-and-disaster-recovery). However, these backups are not publicly accessible because they are stored in a separate system. You can open a ticket to request a backup with a specific date, time, and timezone. Then we can extract it from the external system and provide it to you.
 
+{:.bs-callout-warning}
```

Comment:
This suggestion is optional, but what if we present this as a tip? There is a suggestion here about setting up a cron job and I think it's a really good candidate for a positive Tip with a green checkmark.

```suggestion
{:.bs-callout-tip}
```